users/danielo515/danielo515.h

```diff
@@ -0,0 +1,183 @@
+#ifndef USERSPACE
+#define USERSPACE
+
+#include "quantum.h"
+
+//**************** KEYCODES *********************//
+
+enum custom_keycodes
+{
+  PLACEHOLDER = SAFE_RANGE, // can always be here
+  EPRM,
+  RGB_SLD,
+  ALT_TAB,
+  // Macros
+  ARROW,
+  F_ARROW,
+  QWERTY,
+  GREP,
+  // Accented characters
+  AC_A,
+  AC_E,
+  AC_I,
+  AC_O,
+  // Custom multi-os key-codes
+  CUT,
+  COPY,
+  PASTE,
+  SAVE,
+  UNDO,
+  CHG_LAYOUT,
+  FIND,
+  // OTHER OLD STUFF
+  LOWER,
+  RAISE,
+  ADJUST,
+  MAC_TGL,
+  MACRO_START,  // START OF VSC DECLARATIONS
+  T_TERM,
+  FIX_ALL,
+  BLK_CMNT,
+  LN_CMNT,
+  CMD_S_P,
+  TRI_TICKS,
+  MACRO_END,  // END OF VSC DECLARATIONS
+  INC_MACROS_START,
+  INC_MACROS_END,
+};
+
+//**************** TAP DANCE *********************//
+#define KC_E_COLN LSFT(KC_DOT)
+#define KC_E_EQL ES_EQL
+#define KC_GUI OSM(MOD_RGUI)
+#define KC_R_NUB S(KC_NUBS)
+#define KC_E_LT KC_NUBS
+#define KC_E_GT S(KC_NUBS)
+#define KC_E_TILD ES_TILD
+#define KC_E_MINS ES_MINS
+#define KC_S_SPC SFT_T(KC_SPC)  // Tap for Space, hold for Shift
+#define KC_E_OVRR ES_OVRR
+#define KC_E_APOS ES_APOS
+#define KC_E_IEXL ES_IEXL
+// TAP DANCE KEYS
+#define TD_COPY TD(_TD_COPY)
+#define TD_CUT TD(_TD_CUT)
+#define KC_TD_COPY TD(_TD_COPY) // Declarations for macros that add KC_
+#define KC_TD_CUT TD(_TD_CUT)
+#define KC_TD_PASTE TD(_TD_PASTE)
+#define KC_E_GRV ES_GRV
+
+enum layers {
+  _QWERTY = 0,
```

Comment:
```suggestion
  _QWERTY,
```

You can remove the ` = 0` if you want, it’s implied.